doc/users/next_whats_new/colorbar_position.rst

```diff
@@ -0,0 +1,5 @@
+Gridspec-based colorbars can now be positioned above or to the right of the main axes
```

Comment:
Right, or left? I'm confused by the title vs the argument value below.

Reply:
left, sorry...